.link-checker.js

```diff
@@ -1,3 +1,24 @@
+const HTMLParser = require('node-html-parser');
+
+const getAttributeValue = (obj, attribute) => {
```

Comment:
Could you add a brief comment here on what this does for future context? Something like:

`// Checks object for attribute and returns value. If not found on first pass, recursively checks nested objects and arrays of nested object(s) until attribute is found. If not found, returns undefined.`

Reply:
Added